### PR TITLE
OMERO.Matlab java bindings : Bug fixes, transforms and tables content

### PR DESCRIPTION
**Pull request by bramalingam**
Update the documentation page for OMERO.Matalb bindings.

@sbesson @jburel

### CODE REVIEW

File: omero/developers/Matlab.txt [hunk 1]
```diff
@@ -774,7 +774,7 @@ How to use OMERO tables
 -----------------------
 
 -  **Create a table**. In the following example, a table is created with
-   2 columns.
+   2 columns and is linked to an Image
```
Comment (jburel) [inline omero/developers/Matlab.txt]: ``.`` missing

File: omero/developers/Matlab.txt [hunk 2]
```diff
@@ -798,6 +798,15 @@ How to use OMERO tables
     table.addData(data);
     file = table.getOriginalFile(); % if you need to interact with the table
 
+    % link table to an Image
+    fa = omero.model.FileAnnotationI;
+    fa.setFile(file);
+    % Currently OMERO.tables are displayed only for Screen/plate/wells
```
Comment (jburel) [inline omero/developers/Matlab.txt]: Web client only

File: omero/developers/Matlab.txt [hunk 3]
```diff
@@ -798,6 +798,15 @@ How to use OMERO tables
     table.addData(data);
     file = table.getOriginalFile(); % if you need to interact with the table
 
+    % link table to an Image
+    fa = omero.model.FileAnnotationI;
+    fa.setFile(file);
+    % Currently OMERO.tables are displayed only in the web-client and
```
Comment (jburel) [inline omero/developers/Matlab.txt]: OMERO.web

File: omero/developers/Matlab.txt [hunk 4]
```diff
@@ -898,6 +907,61 @@ attaches it to an image::
...
+    newTform = omero.model.AffineTransformI;
+    newTform.setA00(rdouble(cos(theta)));
+    newTform.setA10(rdouble(-sin(theta)));
+    newTform.setA01(rdouble(sin(theta)));
+    newTform.setA11(rdouble(cos(theta)));
+    newTform.setA02(rdouble(0));
+    newTform.setA12(rdouble(0));
+    % apply transform
+    ellipse.setTransform(newTform);
+    % Create the roi.
```
Comment (jburel) [inline omero/developers/Matlab.txt]: typo no ``.`` to be in synch with the other comments

Comment (hflynn) [reply omero/developers/Matlab.txt]: Also, if this is just added comment, could we have roi written as ROI please?

File: omero/developers/Matlab.txt [hunk 5]
```diff
@@ -798,6 +798,15 @@ How to use OMERO tables
     table.addData(data);
     file = table.getOriginalFile(); % if you need to interact with the table
 
+    % link table to an Image
+    fa = omero.model.FileAnnotationI;
+    fa.setFile(file);
+    % Currently OMERO.tables are displayed only in the OMERO.web and
```
Comment (jburel) [inline omero/developers/Matlab.txt]: drop ``the`` in front of  ``OMERO.web``

**Comment by jburel** [comment]
code PR https://github.com/openmicroscopy/openmicroscopy/pull/5116

File: omero/developers/Matlab.txt [hunk 6]
```diff
@@ -798,6 +798,15 @@ How to use OMERO tables
     table.addData(data);
     file = table.getOriginalFile(); % if you need to interact with the table
 
+    % link table to an Image
+    fa = omero.model.FileAnnotationI;
+    fa.setFile(file);
+    % Currently OMERO.tables are displayed only in OMERO.web and 
+    % for Screen/plate/wells alone.In all cases the file annotation
```
Comment (jburel) [inline omero/developers/Matlab.txt]: minor: space before ``In all cases``

**Comment by jburel** [comment]
Thanks for the matlab version clarification

**Pull request merged**